### ISSUE TITLE
Elevated circulating L-pyroglutamic acid concentration

### ISSUE THREAD

**Issue by JillMiller** [obophenotype/human-phenotype-ontology#6025]
**Preferred term label:** Elevated circulating L-pyroglutamic acid concentration

**Synonyms** Increased level of 5-oxo-L-proline in blood, Elevated circulating 5-oxoproline concentration

**Definition (free text, please give PubMed ID)** Concentration of L-pyroglutamic acid in the blood is above the normal range., PMID: 8789605

**Parent term (use hpo.jax.org/app)** Abnormal circulating glutamate family amino acid concentration; HP:0010902

**Diseases characterized by this term ? (e.g. Orphanet or OMIM number)**

**Your nano-attribution (ORCID)**

**Comment by pnrobinson** [comment]
added term as HP_0033175
primary label  Elevated circulating 5-oxoproline concentration
